linera-sdk/src/contract/conversions_from_wit.rs

```diff
@@ -49,12 +49,11 @@ impl From<wit_system_api::Blob> for Blob {
     }
 }
 
-impl From<wit_system_api::HashedBlob> for HashedBlob {
-    fn from(hashed_blob: wit_system_api::HashedBlob) -> Self {
-        HashedBlob {
-            id: hashed_blob.id.into(),
-            blob: hashed_blob.blob.into(),
-        }
```

Comment:
If `new_unchecked` is kept, why is the `From` eliminated?

Reply:
Because we want this conversion to be checked, which means we need to use `new_checked`, which needs to be `TryFrom` because it returns an error